TESTING.md

```diff
@@ -1,350 +1,352 @@
-    </ol>
-  </div>
-</details>
-<details>
-  <summary>
-    <a href="https://github.com/Aalto-LeTech/intellij-plugin/issues/93">reset to "A+" settings #93</a>
-  </summary>
-  <div>
-    <h5>Resetting A+ Courses Settings</h5>
-    <ol>
-      <li>Start the REPL for a module and check the "Don't show this window again" checkbox.</li>
-      <li>Start the REPL for a module again and ensure that the window isn't shown again.</li>
-      <li>From the A+ menu at the top, select "Reset A+ Courses Plugin Settings".</li>
-      <li>Start the REPL for a module again and observe that the window is shown again.</li>
-    </ol>
-  </div>
-</details>
-<details>
-  <summary>
-    <a href="https://github.com/Aalto-LeTech/intellij-plugin/issues/130">
-      Add AUTO_SHOW_ERRORS_IN_EDITOR="false" to workspace.xml when project settings are imported #130
-    </a>
-  </summary>
-  <div>
-    <h5>Importing project settings</h5>
-    <ol>
-      <li>Use the "Turn Project Into A+ Course Project" menu item to import project settings.</li>
-      <li>Open the file <code>.idea/workspace.xml</code>.</li>
-      <li>
-        Ensure that it contains the following:
-        <pre>
-&lt;component name="CompilerWorkspaceConfiguration">
-  &lt;option name="AUTO_SHOW_ERRORS_IN_EDITOR" value="false" />
-&lt;/component></pre>
-      </li>
-      <li>Use the "Turn Project Into A+ Course Project" menu item again.</li>
-      <li>
-        If <code>workspace.xml</code> now contains option entry multiple times, save the
-        <code>workspace.xml</code> file and ensure that the duplicate gets removed.
-      </li>
-    </ol>
-  </div>
-</details>
-<details>
-  <summary>
-    <a href="https://github.com/Aalto-LeTech/intellij-plugin/issues/182">
-      Cancelling IDE settings import does not work #182
-    </a>
-  </summary>
-  <div>
-    <h5>Course Project Action Dialog</h5>
-    <ol>
-      <li>From the A+ menu select "Turn Project Into A+ Course Project".</li>
-      <li>Observe that the dialog mentions the course name "O1".</li>
-      <li>Observe that by default the restart checkbox is checked and the opt out checkbox isn't.</li>
-      <li>Observe that checking the opt out checkbox disables the restart checkbox.</li>
-      <li>Select "Cancel" and observe that nothing has changed in the project.</li>
-      <li>Select "Turn Project Into A+ Course Project" again.</li>
-      <li>Select "OK" with the default selections and observe that the IDE restarts.</li>
-      <li>Select "Turn Project Into A+ Course Project" again.</li>
-      <li>Observe that the dialog mentions that IDE settings are already imported for O1.</li>
-      <li>Observe that the restart checkbox is unchecked and not enabled.</li>
-      <li>Observe that the opt out checkbox is checked and not enabled.</li>
-    </ol>
-  </div>
-</details>
+Manual testing
+==============
+
+[The complete list](https://github.com/Aalto-LeTech/intellij-plugin/labels/manual%20testing)
```

Comment:
I suggest a minor improvement here: https://github.com/Aalto-LeTech/intellij-plugin/issues?q=label%3A%22manual+testing%22+ , otherwise "by default" filter shows only the **"open"** issues